src/holdingpen/containers/DashboardPage.jsx

```diff
@@ -1,10 +1,65 @@
 import React, { Component } from 'react';
+import data from '../report-errors.json';
+import CategoryCard from '../components/CategoryCard';
+import ExceptionsTable from '../components/ExceptionsTable';
+import { Row, Col } from 'antd';
 
 class DashboardPage extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      exceptions: this.mapData(data),
+      collections: this.calculateExceptionsMetadata('collection')
+    }
+  }
+
+  mapData = (data) => {
+    const rawData = [...data];
+    const exceptions = rawData.map( (exception, index) => {
+      const keys = Object.keys(exception)
+      return ({
+        "key": index,
```

Comment:
no doublequotes,

I am not commenting the other mistakes like this since you should run `eslint` locally `yarn lint` before pushing and fix mistakes. I'd suggest using an extension to see errors as you write your code.